sys/kern/klog.c

```diff
@@ -43,8 +43,7 @@ static const char *subsystems[] = {
 };
 
 void init_klog(void) {
```

Comment:
Why not initialize klog structure statically? Do we really need code for doing that ?

Reply:
You're right! Done.